extensions/airship/airship-event-extension.schema.json

```diff
@@ -0,0 +1,271 @@
+{
+  "$id": "https://ns.airship.com/airship-event-extension",
+  "title": "Airship Event Extension",
+  "description": "This mixin adds Airship specific event fields.",
+  "meta:intendedToExtend": [
```

Comment:
Need to add a $schema attribute here. Look at other schemas in this repo and copy it.